client/src/helpers/Modal.js

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react'
+import ModalContext from '../contexts/ModalContext'
+
+
+export default () => {
+  const {
+    modalIsOpen,
+    setModalIsOpen,
+    headingText,
+    setHeadingText,
+    contentText,
+    setContentText
+  } = useContext(ModalContext)
+
+  if (!modalIsOpen) {
+    return <div />
+  }
```

Comment:
It might be better to not render a div at all.  Two possible solutions below.  I would also recommend moving this conditional below the handle click.  That way it is closer to the other return statement.

Null:
```javascript
if (!modalIsOpen) {
  return null
}
```

Fragment
```javascript
if (!modalIsOpen) {
  return <></>
}
```